### ISSUE TITLE
v0.0.8 fails to build

### ISSUE THREAD

**Issue by nim65s** [stack-of-tasks/roscontrol_sot#6]
http://robotpkg.openrobots.org/rbulk/robotpkg-wip/wip/roscontrol-sot/roscontrol-sot-0.0.8/Ubuntu-16.04-x86_64/build.html

```
[…]/roscontrol_sot-0.0.8/src/roscontrol-sot-controller.cpp:765:55: error: 'class hardware_interface::JointHandle' has no member named 'getAbsolutePosition'
DataOneIter_.joint_angle[idJoint] = joints_[idJoint].getAbsolutePosition(); 
```

**Comment by olivier-stasse** [comment]
Dear nim65s,
This is due to an option inside jrl-cmakemodules which uses the -isystem option for includes by default. This breaks the chained workspace integration and forbid to have the PAL extensions. There is already a patch if you compile through robotpkg (or use the binary).

The problem appeared during the last rollout of OSRF with new versions of ros packages. I am just wondering why it did not appear before.

The jrl-cmakemodules needs a careful update as it is likely that a modification might have a strong impact on other packages.

**Comment by nim65s** [comment]
Thanks for the investigation and the quickfix !
I will have a look at it more deeply in the cmake module asap.

**Comment by olivier-stasse** [comment]
Fixed by 4a07ffbba9bb8cc6cea798fd657f8ba2b46a0249
